client/pages/Invite/index.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+import moment from 'moment-timezone';
+
+import { validateEventInvite } from '../../actions/graphql/eventInviteGQLActions';
+import '../../assets/pages/_event_invite-page.scss';
+
+
+class InvitePage extends Component {
+  // Steps:
+  //   1. Validate Invite ID
+  //   2. Either display appropriate error or redirect to event
+  componentDidMount() {
+    // Validate Invite ID,
+    const {
+      match: { params: { inviteHash } }, validateInviteAction,
+    } = this.props;
+    validateInviteAction(inviteHash);
+  }
+
+  render() {
+    const {
+      invite: {
+        isValid,
+        message,
+      },
+    } = this.props;
+    if (isValid) {
+      const {
+        invite: {
+          event: {
+            id, featuredImage, title, venue, startDate,
+          },
+        },
+      } = this.props;
+      return (
+        <React.Fragment>
+          <div className="event-invite__main_card">
+            <p className="event-invite__title">You have been invited to attend {title}.</p>
+            <img className="event-invite__event_img" src={featuredImage} alt="event img"/>
+            <p className="event-invite__title">Venue:  {venue}.</p>
+            <p className="event-invite__title">Starts On:  {moment.tz(startDate, moment.tz.guess()).format('llll')}.</p>
+            <NavLink
+              to={`/events/${id}`}
+              className="event-invite__button event-invite__go_btn"
+              >
+              View Event Details
+            </NavLink>
+          </div>
+        </React.Fragment>
+      );
+    }
+    return (
+      <React.Fragment>
+        <div className="event-invite__main_card">
+          <p className="event-invite__error">Sorry {message}.</p>
+        </div>
+      </React.Fragment >
+    );
+  }
+}
+
+InvitePage.propTypes = {
+  match: PropTypes.shape(
+    {
+      params: PropTypes.shape(
+        { inviteHash: PropTypes.string }
+      ),
+    }
+  ).isRequired,
+  invite: PropTypes.shape({}).isRequired,
```

Comment:
is the shape of invite always
```
invite: {
   isValid,
   message,
}
```
or the shape is not always the same.

Reply:
You can use `object` instead `shape` then

Reply:
@kenneth-kip es-lint forbids the use of proptype `object`. 
<img width="509" alt="screen shot 2018-10-05 at 10 54 56 am" src="https://user-images.githubusercontent.com/19896027/46523023-2286be80-c88d-11e8-9de9-800cc35ff4fb.png">

Reply:
Ah! ok.